### PR TITLE
bpo-46184: remove `netlify.toml`

### PR DESCRIPTION
**Pull request by sobolevn**
Related to https://github.com/python/cpython/pull/30257

I guess we can skip NEWS on this one.

https://bugs.python.org/issue46184

### CODE REVIEW

**Comment by sobolevn** [comment]
Windows build for `winarm` take a lot of time, then it is canceled. https://dev.azure.com/Python/cpython/_build/results?buildId=93679&view=logs&j=91c152bd-7320-5194-b252-1404e56e2478

This failure does not seem related.

**Review by kumaraditya303 — APPROVED** [review]
LGTM as it seems unused.

**Comment by hugovk** [comment]
This file was added in https://github.com/python/cpython/pull/15288 / https://bugs.python.org/issue37860 but wasn't turned on due to build limits:

https://github.com/python/cpython/pull/15288#issuecomment-579476340

I guess that was never resolved, so okay to remove this file, @epicfaace? 

If so, should also remove `Doc/runtime.txt`, added for Netlify in the same PR. 

And see https://github.com/python/cpython/pull/19025 for an open, alternate approach using GitHub Actions.

File: Doc/requirements.txt [hunk 1]
```diff
@@ -1,12 +0,0 @@
-# Requirements to build the Python documentation
```
Comment (hugovk) [inline Doc/requirements.txt]: `Doc/requirements.txt` is still needed:

https://github.com/python/cpython/runs/4649987492?check_suite_focus=true#step:7:30

Comment (sobolevn) [reply Doc/requirements.txt]: @hugovk thank you! 👍

**Comment by sobolevn** [comment]
@AlexWaygood do you think that this can be marked as "skip news"?

**Comment by AlexWaygood** [comment]
> @AlexWaygood do you think that this can be marked as "skip news"?

Okay, I am adding the skip-news label,  as it seems appropriate to me. But, this is not something I really know anything about, so anybody who thinks that this needs a news entry should feel free to remove it again. 🙂

**Comment by epicfaace** [comment]
> I guess that was never resolved, so okay to remove this file, @epicfaace?

Yes, sgtm!

**Comment by sobolevn** [comment]
@Mariatta are you the right person to ping in this case? 🙂

**Review by Mariatta — APPROVED** [review]
Thanks for cleaning these up.

**Pull request merged**